package.json

```diff
@@ -48,6 +48,7 @@
     "prepare": "git config core.hooksPath .githooks",
     "eslint": "npx eslint \"**/*.ts\" \"**/*.tsx\"",
     "eslint-fix": "npx eslint \"**/*.ts\" \"**/*.tsx\" --fix",
+    "typescript": "npx -p typescript tsc -b",
```

Comment:
Is this necessary, since there's already a build command?

Reply:
Nah, not really! I just did it as a shortcut without starting a commit, because I forgot the command.

Reply:
Could remove before merge

Reply:
Accepting merge now; if need be this can be removed later :+1: